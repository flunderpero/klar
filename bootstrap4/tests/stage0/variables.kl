fn main():
    let a = 1
    let b = 2
    print(a)
    print(b)
end
