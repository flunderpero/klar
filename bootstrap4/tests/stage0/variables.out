1
2
---
exit code: 0
